README.md

```diff
@@ -870,55 +1080,79 @@ Creates a database with no users and no permissions.
 
 ##### `dbname`
 
-Sets the name of the database. Defaults to the namevar.
+Sets the name of the database. 
+
+Defaults value: The namevar.
 
 ##### `encoding`
 
-Overrides the character set during creation of the database. Default: The default defined during installation.
+Overrides the character set during creation of the database. 
+
+Default value: The default defined during installation.
 
 ##### `istemplate`
 
-Defines the database as a template if set to true. Default: false.
+Defines the database as a template if set to `true`.
+
+Default value: `false`.
 
 ##### `locale`
 
-Overrides the locale during creation of the database. The default defined during installation.
+Overrides the locale during creation of the database.
+
+Default value: The default defined during installation.
 
 ##### `owner`
 
-Sets name of the database owner. Default: The $user variable set in `postgresql::server` or `postgresql::globals`.
+Sets name of the database owner.
+
+Default value: The '$user' variable set in `postgresql::server` or `postgresql::globals`.
 
 ##### `tablespace`
 
-Sets tablespace for where to create this database. Default: The defaults defined during PostgreSQL installation.
+Sets tablespace for where to create this database. 
+
+Default value: The default defined during installation.
 
 ##### `template`
 
-Specifies the name of the template database from which to build this database. Default: `template0`.
+Specifies the name of the template database from which to build this database. 
+
+Default value: 'template0'.
 
 #### postgresql::server::database_grant
 
 Manages grant-based access privileges for users, wrapping the `postgresql::server::database_grant` for database specific permissions. Consult the [PostgreSQL documentation for `grant`](http://www.postgresql.org/docs/current/static/sql-grant.html) for more information.
 
 #### `connect_settings`
 
-Specifies a hash of environment variables used when connecting to a remote server. Default: Connects to the local Postgres instance.
+Specifies a hash of environment variables used when connecting to a remote server. 
+
+Default value: Connects to the local Postgres instance.
 
 ##### `db`
 
 Specifies the database to which you are granting access.
 
 ##### `privilege`
 
-Specifies comma-separated list of privileges to grant. Valid options: `ALL`, `CREATE`, `CONNECT`, `TEMPORARY`, `TEMP`.
+Specifies comma-separated list of privileges to grant. 
+
+Valid options: 'ALL', 'CREATE', 'CONNECT', 'TEMPORARY', 'TEMP'.
 
 ##### `psql_db`
 
-Defines the database to execute the grant against. **This should not ordinarily be changed from the default**, which is `postgres`.
+Defines the database to execute the grant against. 
+
+Default value: 'postgres'.
+
+**This should not ordinarily be changed from the default**
```

Comment:
I would move this warning sentence up above the Default value, rather than after it.